pydruid/client.py

```diff
@@ -120,7 +120,7 @@ def __post(self, query):
             headers = {'Content-Type': 'application/json'}
             req = urllib.request.Request(url, querystr, headers)
             res = urllib.request.urlopen(req)
-            data = res.read()
+            data = res.read().decode()
```

Comment:
default for `bytes.decode` is utf-8 ( https://docs.python.org/3/library/stdtypes.html#bytes.decode ).
But I just updated to make it explicit.